src/ResourceManager/AnalysisServices/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added State property in additional to ProvisioningState
```

Comment:
@markcowl , I have added more information about this change.
Currently ASPAAS service still return same for both State and ProvisioningState, we could not tell the user the actual different example until service side make change in future.

I already provided the difference in intention